prow/apis/prowjobs/v1/types.go

```diff
@@ -108,10 +108,10 @@ type ProwJobSpec struct {
 	Job string `json:"job,omitempty"`
 	// Refs is the code under test, determined at
 	// runtime by Prow itself
-	Refs *Refs `json:"refs,omitempty"`
+	Refs Refs `json:"refs,omitempty"`
```

Comment:
Unfortunately this actually should remain as `*Refs`. We want to distinguish unset from unconfigured (and we do not want every prow job to always output the Refs field.